lib/dartino.dart

```diff
@@ -103,6 +105,48 @@ void openDartinoSettings([_]) {
   atom.workspace.open('atom://config/packages/dartino');
 }
 
+_createNewProject([_]) async {
+  var sdk = await findSdk(null);
+  if (sdk == null) return;
```

Comment:
Perhaps `atom.notifications.addError('...')`? Not sure when `sdk` would be null, but perhaps worth notifying the user about?